clang-tools-extra/clangd/ConfigCompile.cpp

```diff
@@ -622,6 +622,31 @@ struct FragmentCompiler {
             C.Completion.AllScopes = AllScopes;
           });
     }
+    if (F.ArgumentLists) {
```

Comment:
We have a `compileEnum` helper to do this more easily, see [the handling of `BackgroundPolicy`](https://searchfox.org/llvm/rev/87d56c59f52d033cd7c46d769338b9c47fea4929/clang-tools-extra/clangd/ConfigCompile.cpp#326-334) as an example